<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drawMainWindowClass</class>
 <widget class="QMainWindow" name="drawMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>drawMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/baiduMusicPlayerPNG/default_lyric_artist_pic.png);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_readData">
    <property name="title">
     <string>读取数据(R)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_displayData">
    <property name="title">
     <string>显示数据(&amp;D)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Command">
    <property name="title">
     <string>控制命令面板(&amp;C)</string>
    </property>
    <addaction name="actionSetSysParameter"/>
    <addaction name="actionTestCommandDialog"/>
   </widget>
   <addaction name="menu_readData"/>
   <addaction name="menu_displayData"/>
   <addaction name="menu_Command"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetSysParameter">
   <property name="text">
    <string>设置系统参数</string>
   </property>
  </action>
  <action name="actionTestCommandDialog">
   <property name="text">
    <string>测试命令面板</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="drawmainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
